WordPressCom-Stats-iOS/Private/ExtensionUtils.m

```diff
@@ -0,0 +1,25 @@
+#import "ExtensionUtils.h"
+
+@implementation ExtensionUtils
+
```

Comment:
This looks good, in that I think it'll work and putting it in an extension is great. Only thing I'd like to see is a comment to explain why we've done this, in the hopes that at some future date we can remove it.